### DISCUSSION TITLE
newbie needs help

### DISCUSSION THREAD

**Discussion by Maligola** [rern/rAudio#463] [General]
Hi I am coming from an old (2018) Volumio version which unfortunately no longer runs cleanly and the old plugins no longer run with the current Volumio versions. Therefore I am looking for a replacement and came up here.
However, some things don't run as expected:

Checked with: Raspberry Pi Zero and pi 3b
Software: rAudio 20210716  for Zero and pi3-4 Version

Problems:
1. the device/software forgets the wifi password after the first reboot, when I upload the wifi file again it works exactly once again. Is there another place where I can enter the data permanently?
Completely reinstalled several times. 
LAN works; Wlan does not.

2. The 16x2 display only shows me the title of the track in the first line.  However, no artists or similar. 
The second line is not used.
Can this be set somewhere?

3. there is no online radio database that can be accessed directly from the software? Similar to volumio? Do I have to enter all the radio stream links individually?

--> if you want me to post my question in another forum, please let me know Where?

Thx

**Comment by R101G** [comment]
If you are using the /boot/wifi file to connect the first time, there should not be a problem. You can check in networks settings to see what has been saved. If it connects OK the first time, there is no obvious reason why it should not connect again unless the connection is weak. You can check the connection quality from the networks settings page, under the twin cogs symbol. 

If you are using DHCP, are you sure the router is allocating the same IP address every time?

**Reply by Maligola** [reply]
Unfortunately, this is a non-reproducible problem.

I have created the Wifi file and transferred it to the SD card.

The Pi is booted initially and there is a Wifi connection. 
In the router, the DHCP IP is assigned permanently/fixed.

Now I have to restart the device because I have activated the display.


After the restart, a wireless connection is no longer possible. The router no longer shows the device as listed. Also checked with the mobile phone wifi scan.

I took the SD card out of the Raspberry and looked at the boot directory, the wifi file is gone. 


--> Re-created the wifi file and then it works exactly 1 time again with wifi reception. after that the wifi file is gone again.

---


Therefore the question if there is another place for the WIFI data?

**Comment by R101G** [comment]
I am afraid you will have to apply some logic to solve this. Does it reconnect if you reboot without activating the local display?

Try reading my previous reply. The wifi settings file is in /etc/netctl/
